tests/unit/lms/views/lti/basic_launch_test.py

```diff
@@ -340,12 +336,6 @@ def test__show_document_does_not_enable_instructor_toolbar_for_students(
 
         context.js_config.enable_instructor_toolbar.assert_not_called()
 
-    @pytest.mark.usefixtures("user_is_instructor", "with_non_gradable_assignment")
```

Comment:
This is now part of the parameterized test (and the behavior is to show the bar)